src/components/styles/_glow.less

```diff
@@ -1,88 +1,51 @@
-.glowBorderRule(@color) {
-  border-color: @color;
-  background-color: rgba(@color, 0.2);
-}
-
 .glow-btn {
-  transition: all 0.5s ease;
   border: 2px solid;
   color: @black;
```

Comment:
Can you change this to `@darkText !important;` ? I know it was like this in the original code but it would be a good fix to it.

Reply:
Done